.github/markets/pr_template.md

```diff
@@ -22,6 +22,7 @@ Each json file under the [configs](../../configs) folder correspond to their res
 |`demex_trading_league_config` |`DemexTradingLeagueConfig` |false |Object that contains the parameters for the current trading league. |
 |`perp_pools`   |`PerpPoolConfig`   |false   |Object that contains the configs for Perp Pools   |
 |`wswth_contract`   |`string`   |false   |wSWTH ERC-20 contract.   |
+|`market_banners`   |`object[]`   |true   |market banner configs.   |
```

Comment:
Pls make sure to define the `MarketBanner` struct in this Markdown file, like [this](https://github.com/Switcheo/demex-webapp-config/blob/feat/market-banners/.github/markets/pr_template.md#perppoolbanner).